frontend/fingertips-frontend/app/chart/page.tsx

```diff
@@ -35,15 +35,17 @@ export default async function ChartPage(
   try {
     rawPopulationData = await indicatorApi.getHealthDataForAnIndicator({
       indicatorId: indicatorIdForPopulation,
-      areaCodes: [areaCodes[0], areaCodeForEngland],
+      areaCodes: [...areaCodes, areaCodeForEngland],
     });
   } catch (error) {
     console.log('error getting population data ', error);
   }
 
   if (rawPopulationData) {
-    // hardcode selected area data for mocks while no population data are in dbase
-    preparedPopulationData = preparePopulationData(rawPopulationData, '1', '2');
+    preparedPopulationData = preparePopulationData(
+      rawPopulationData,
+      ...areaCodes
+    );
```

Comment:
I'm not sure it necessarily makes sense to just plug all selected area codes in here. Especially since the `preparePopulationData` function only expects to take the area code for a single selected area and the baseline. We'd ultimately want to call this function in a loop (or refactor it to take a list of selected areas), but that's definitely not part of this ticket.

I think it might be more consistent to just pass it the first area code for now (with a comment explaining):
```suggestion
    preparedPopulationData = preparePopulationData(
      rawPopulationData,
      areaCodes[0] // Only passing the first area code until support for pop pyramids for multiple areas is added
    );
```